project-base/UPGRADE.md

```diff
@@ -1,6 +1,14 @@
 # Upgrading
 
 ## From 7.0.0-alpha1 to Unreleased
+- check changes in the `docker-compose.yml` template you used, there were a couple of important changes you need to replicate
+    - easiest way is to overwrite your `docker-compose.yml` with by the appropriate template
+- on *nix systems, fill your UID and GID (you can run `id -u` and `id -g` to obtain them) into Docker build arguments `www_data_uid` and `www_data_gid` and rebuild your image via `docker-compose up --build`
+- change owner of the files in shared volume to `www-data` from the container by running `docker exec -u root shopsys-framework-php-fpm chown -R www-data /var/www/html`
+    - the user has shared UID, so you will be able to access it as well from the host machine
+    - shared volume with postgres data should be owned by `postgres` user: `docker exec -u root shopsys-framework-php-fpm chown -R postgres /var/www/html/var/postgres-data`

```

Comment:
the `postgres-data` folder does not exist on Windows so this instruction is for *nix only too? I was confused when trying to follow the guidelines here